Dockerfiles/midstream/fix_etc_passwd.sh

```diff
@@ -1,25 +1,33 @@
-#!/bin/bash -e
+#!/usr/bin/bash -e
 main() {
     local username="${SSH_AUTH_USER:-default}"
     local home
     home=$(mk_home "$username")
     setup_passwd "$username" "$home"
+    if [ "$#" -eq 0 ]; then
+      echo "Running inside a openshift environment \n
+            No arguments to script detected.\n
+            sleeping indefintely."
+      exec sleep inf
+    fi
 }
+
 mk_home() {
     local username="${1:?}"
-    mkdir -p "/tmp/home/$username" && \
-    echo "/tmp/home/$username" && \
-    return 0
+    [[ -d /workDir ]] && [[ -w /workDir ]] && \
+        mkdir -p "/workDir/home/$username" && \
+        echo "/workDir/home/$username" && \
+        return 0
```

Comment:
What would happen if `/workDir` does not exist ?
how will we debug that ?

Reply:
New script does not have have /workDir. 
This workDir is expected in midstream.